vspace/src/main/java/edu/asu/diging/vspace/core/model/impl/VideoBlock.java

```diff
@@ -0,0 +1,42 @@
+package edu.asu.diging.vspace.core.model.impl;
+
+import javax.persistence.Entity;
+import javax.persistence.OneToOne;
+
+import edu.asu.diging.vspace.core.model.IVSVideo;
+import edu.asu.diging.vspace.core.model.IVideoBlock;
+
+@Entity
+public class VideoBlock extends ContentBlock implements IVideoBlock {
+    
+    private String url;
+    
+    @OneToOne(targetEntity = VSVideo.class)
+    private IVSVideo video;
```

Comment:
I would make the distinction of local vs remote videos on the VSVideo level and you block would only have the video. This way, if we want to list all videos used in the exhibition we can just show all video objects. The block is really just for showing it, it should capture the business logic of remove vs local.